src/popup/components/TotalWalletAmount.vue

```diff
@@ -10,27 +10,27 @@
 <script lang="ts">
 import { computed, defineComponent } from '@vue/composition-api';
 import BigNumber from 'bignumber.js';
-import { useGetter } from '../../composables';
-import { rxJsObservableToVueState } from '../utils';
+import { useBalances } from '../../composables';
+import { useGetter } from '../../composables/vuex';
 
 export default defineComponent({
   setup(props, { root }) {
-    const balances = rxJsObservableToVueState<BigNumber[]>(
-      (root.$store.state as any).observables.balances,
-    );
+    const { balances } = useBalances({ store: root.$store });
+    const balaX = computed(() => Object.values(balances.value).map((v) => v.toNumber()));
 
     const convertToCurrencyFormatted = useGetter('convertToCurrencyFormatted');
 
     const totalAmount = computed(() => {
-      if (!balances.value?.length) return 0;
-      const total = balances.value.reduce(
+      const total = Object.values(balances.value).reduce(
         (previousValue, currentValue) => previousValue.plus(currentValue),
         new BigNumber(0),
       );
-      return convertToCurrencyFormatted.value(total);
+      return convertToCurrencyFormatted.value(total.toNumber());
     });
 
     return {
+      balances,
+      balaX,
```

Comment:
Is`balaX` variable actually used?

Reply:
My mistake. I was using it for debugging purposes. I'll remove it.